server.js

```diff
@@ -1,46 +1,46 @@
-var express = require('express');
-var fs = require('fs');
-var request = require('request');
-var cheerio = require('cheerio');
-var app = express();
+const express = require('express');
+const fs = require('fs');
+const request = require('request');
+const cheerio = require('cheerio');
+const app = express();
+
+const URL = 'http://www.imdb.com/title/tt5052448/?ref_=fn_al_tt_1'
+const TITLE = '.title_wrapper > h1[itemprop="name"]'
+const RELEASE_DATE = '.title_wrapper > h1[itemprop="name"] a'
 
 
 app.get('/scrape', function(req, res) {
 
-  let url = 'http://www.imdb.com/title/tt5052448/?ref_=fn_al_tt_1'
+  request( URL, function(error, response, html) {
+    if (!error) {
+      const $ = cheerio.load(html);
+      const json = {}
+
+      $( TITLE ).filter(function() {
```

Comment:
I know that the tutorial uses a `filter()`, but if you think about what a [filter does](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter?v=example), I don't know that that's the right tool here (even though it's probably still working). Try `each()`